### PR TITLE
 clean up

### PR DESCRIPTION
**Pull request by iemeka**

### CODE REVIEW

**Review by dokasto — COMMENTED** [review]
Word per minute calculation still needs some work

**Review by dokasto — APPROVED** [review]
alt

File: src/Accuracy.js [hunk 1]
```diff
@@ -1,14 +1,35 @@
...
+import text from "./resources/content";
 import "./Accuracy.css";
 
 export default function Accuracy() {
-  const { numLetterPassed, numLetterFailed,lengthOfText } = useContext(utilityContext);
+  const { pressedKeys } = useContext(utilityContext);
+  const [numLetterFailed, setNumLetterFailed] = useState(0);
+  const [numLetterPassed, setNumLetterPassed] = useState(0);
+
+  const currentIndex = useCallback(
```
Comment (dokasto) [inline src/Accuracy.js:11]: This is a function? why use it as a variable?

Comment (iemeka) [reply src/Accuracy.js:11]: was trying out that syntax

**Pull request merged**